content/projects/gsoc/2019/plugin-installation-manager-tool-cli.adoc

```diff
@@ -48,6 +48,11 @@ image::/images/post-images/gsoc-plugin-management-tool/chucknorrisdetails.png[Pl
 
 image::/images/post-images/gsoc-plugin-management-tool/plugininjenkins.png[Plugin Installed in Jenkins, align="center"]
 
+==== Helpful Links
```

Comment:
+1 for moving this to the top.  I think it makes it a lot easier to see everything you were working on.